portality/api/v1/crud/articles.py

```diff
@@ -42,7 +71,7 @@ def create(cls, data, account):
 
         # Check we are allowed to create an article for this journal
         if not XWalk.is_legitimate_owner(am, account.id):
-            raise Api404Error()                 # not found for this account
+            raise Api403Error()
```

Comment:
I thought it was too confusing to return a 404 for a **related** resource. Like, I send a POST create request and I get a 404 .. what? What wasn't found? A 404 works on retrieve and update and such, but not really on create - should just use 403.